typescript/formsApps.d.ts

```diff
@@ -422,12 +423,17 @@ export type OrganisationAppUser = {
   formsAppIds: number[]
 }
 
-export type FormsAppUserBase = {
+export type NewFormsAppKey = {
+  formsAppId: number
+  groups: string[]
+}
+
+export type FormsAppKey = IdResource & NewFormsAppKey
+
+export type FormsAppUserBase = NewFormsAppKey & {
```

Comment:
Not thrilled about NewFormsAppKey being a subset of FormsAppUserBase (even thought at this stage it is only a name)

Reply:
Suggestion?